slack-plugin/src/main/kotlin/slack/unittest/UnitTests.kt

```diff
@@ -222,13 +222,13 @@ internal object UnitTests {
             else -> project.rootProject.projectDir.absolutePath
           }
         jvmArgs(
-          "-XX:+HeapDumpOnOutOfMemoryError",
+          "-XX:+HeapDumpOnOutOfMemoryError", // Produce a heap dump when an OOM occurs
+          "-XX:+CrashOnOutOfMemoryError", // Produce a crash report when an OOM occurs
```

Comment:
Should you also be adding- ExitOnOutOfMemoryError?

Reply:
Nope, you would pick only one of exit or crash